src/gui/dbsettings/DatabaseSettingsWidgetGeneral.ui

```diff
@@ -190,6 +190,58 @@
         </property>
        </widget>
       </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QCheckBox" name="autosaveDelayCheckBox">
+          <property name="toolTip">
+           <string>Autosave delay since last change</string>
+          </property>
+          <property name="accessibleName">
+           <string/>
+          </property>
+          <property name="text">
+           <string>Autosave delay</string>
```

Comment:
As end user I would see `Autosave every 5 min` rather `Autosave delay...`.

Reply:
Never mind, your version is correct because auto saving is related to the last changes rather regular scheduling.